test/Microsoft.DotNet.Tools.Tests.Utilities/Assertions/CommandResultAssertions.cs

```diff
@@ -42,7 +42,7 @@ public AndConstraint<CommandResultAssertions> Fail()
         public AndConstraint<CommandResultAssertions> HaveStdOut()
         {
             Execute.Assertion.ForCondition(!string.IsNullOrEmpty(_commandResult.StdOut))
-                .FailWith(AppendDiagnosticsTo("Command did not output anything to stdout"));
+                .FailWith(AppendDiagnosticsTo($"Command did not output anything to stdout"));
```

Comment:
nit: Adding `$` doesn't do anything... unless the rare situation that `AppendDiagnosticsTo` takes in [FormattableString](https://learn.microsoft.com/en-us/dotnet/api/system.formattablestring?view=net-8.0).